Makefile

```diff
@@ -45,9 +46,11 @@ tf-init:
 	$(TF_RUNNER) -chdir=$(TF_DIR) init -input=false $(TF_BACKEND_CONFIG)
 
 tf-plan-apply:
+	touch $(TF_DIR)/combined-openapi.json
```

Comment:
had to create an empty file because when running tf-plan rest_api and template_file giving an error if the file doesn't exist